packages/teleport/src/Discover/Discover.test.tsx

```diff
@@ -58,6 +58,8 @@ const fullAcl: Acl = {
   desktopSessionRecordingEnabled: true,
   directorySharingEnabled: true,
   connectionDiagnostic: fullAccess,
+  license: fullAccess,
+  download: fullAccess,
```

Comment:
Nit: You didn't introduce this, but I don't love that we need to update this in 3 places. Since we have an exported const that is the same value in `packages/teleport/src/Main/fixtures/index.ts` why aren't we leveraging that for the two tests?

Reply:
Yeah, makes sense.
Added this here https://github.com/gravitational/webapps/pull/1452/commits/f939601a1d04a21e3be3a510a8c6cfc648b8d6cb